checkers/model_form.py

```diff
@@ -37,24 +42,46 @@ def is_model_form_attribute(self, base):
             base.value.id == 'models' and base.attr == 'ModelForm'
         )
 
+    def is_string_dunder_all(self, element):
+        """
+        Return True if element is ast.Str and equals "__all__"
+        """
+        return (
+            isinstance(element.value, ast.Str) and
+            element.value.s == '__all__'
+        )
+
     def run(self, node):
         """
         Captures the use of exclude in ModelForm Meta
         """
         if not self.checker_applies(node):
             return
 
+        issues = []
         for body in node.body:
             if not isinstance(body, ast.ClassDef):
                 continue
             for element in body.body:
+                is_fields = False
                 if not isinstance(element, ast.Assign):
                     continue
                 for target in element.targets:
-                    if target.id == 'exclude':
-                        return [
+                    if target.id == 'fields':
```

Comment:
What about:

```python
if target.id == 'fields' and self.is_string_dunder_all(element):
    issues.append(
        DJ07(
            lineno=node.lineno,
            col=node.col_offset,
        )
    )
```

Reply:
thanks for the suggestion :) much better like this!